docs/trendz/install/ubuntu.md

```diff
@@ -136,7 +136,7 @@ sudo service trendz start
 Once started, you will be able to open Web UI using the following link:
 
 ```bash
-http://localhost:8888/
+http://localhost:8888/trendz
```

Comment:
This change is not backward-compatible?

Reply:
it is backward-compatible, but for cases when Trendz and TB installed on the same server and works under same domain - user should use /trendz prefix